registration_server.py

```diff
@@ -18,15 +18,33 @@ class Participant(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     first_name = db.Column(db.String(100))
     last_name  = db.Column(db.String(100))
+    pronoun = db.Column(db.String(100))
     email = db.Column(db.String(100))
     affiliation = db.Column(db.String(100))
-    lname = db.Column(db.String(100))
-    sname = db.Column(db.String(100))
-    pronoun = db.Column(db.String(100))
+    #lname = db.Column(db.String(100))
+    #sname = db.Column(db.String(100))
     contact = db.Column(db.String(5))
     volunteer = db.Column(db.String(5))
     volunteer_responsibilities = db.Column(db.String(200))
     # visa    = db.Column(db.String(5))
+    CL = db.Column(db.String(5))
```

Comment:
If you are not usisng  these fields anymore, better to remove them

Reply:
Right, thank you!